techniques/general/G219-prefil.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+	<head>
+		<title>Prefill an input with the data from a previous answer</title>
+		<link rel="stylesheet" type="text/css" href="../../css/editors.css"/>
+	</head>
+	<body>
+		<h1>Prefill an input with the data from a previous answer</h1>
+		<section id="meta">
+			<h2>Metadata</h2>
+			<p class="instructions">Provide information below to help editors associate the technique properly. Contents of the meta section are not output by the processor.</p>
+			<p id="sc">3.3.8 Redundant entry</p>
+			<p id="type">Sufficient</p>
+		</section>
+		<section id="applicability">
+			<h2>Applicability</h2>
+			<p class="instructions">Any technology that supports scripting.</p>
+		</section>
+		<section id="description">
+			<h2>Description</h2>
+			<p>The objective of this technique is to prefill data entry fields with any information that was previously provided by the user or by the system rather than requiring the user to remember and re-enter the information from a previous step. This improves success for individuals with cognitive and learning disabilities and memory impairments.</p>
+
+			<p>This technique best applies when information is often, but not always, the same as information previously entered or provided.</p>
+				
+
+		</section>
+		<section id="examples">
+			<h2>Examples</h2>
+			<section class="example">
+				<h3>Populate previously entered information with a trigger</h3>
+				<p>An ecommerce site provides a tick-box that triggers the shipping address to be pre-populated based on the billing address the user had entered in a previous step. </p>
+
+			</section>
+			<section class="example">
+				<h3>Automatically populate information based on previous answers</h3>
+				<p>During a two step process for setting up a business, the first step provides a business identification number after the user enters a name and address. The second step requires creating a user profile. The business identification number, name and address are prepopulated. </p>
+			</section>
+		</section>
+		<section id="tests">
+			<h2>Tests</h2>
+			<section class="test-procedure">
+				<h3>Procedure</h3>
+				<p>For each input in a process that requires information to be entered:</p>
+				<ol>
```

Comment:
This test process doesn't talk about a previous step, so a "confirm the email address" pattern would fail this technique.